### PR TITLE
[V:1.1.0] chore(requirements): support python 3.8

### PR DESCRIPTION
**Pull request by elliottyates**
- fixes to test files to ensure that code can run under python 3.8
- config updates to indicate production status and support python 3.8
- drop support for python 3.5 (end of life Sep 2020)
- version bump to 1.1.0
- copyright bump

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 penzance
+Copyright (c) 2020 penzance
```
Comment (cmurtaugh) [inline LICENSE]: [minor] my inclination would be to change `penzance` to `President and Fellows of Harvard College` here.

Comment (elliottyates) [reply LICENSE]: Done! Thanks for the suggestion!

**Comment by elliottyates** [comment]
FYI I will squash these commits and re-tag `1.1.0` after receiving at least one approval.

**Comment by arthurian** [comment]
Looks good to me! :+1:

**Pull request merged**